ui/tests/acceptance/jobs-list-test.js

```diff
@@ -4,7 +4,12 @@
  */
 
 /* eslint-disable qunit/require-expect */
-import { currentURL, click, triggerKeyEvent } from '@ember/test-helpers';
+import {
```

Comment:
Generally: added a bunch of tests, and un-skipped the facet and other search/filter tests.

Notable omissions:
1. things that tested manual sorting (we no longer allow users to sort) although we have a new "are they still sorted by ModifyIndex desc" test
2. things to do with Prefix and DataCenter dropdown filtering


Almost every test had something like "the currentURL has a literal representation of what you filtered in it", and that is now "the currentURL has a filter expression that corresponds to what you filtered in it"